playbook/app/pb_kits/playbook/pb_background/_background.scss

```diff
@@ -22,14 +22,30 @@ $background_colors: map-merge($additional_colors, $merge_kits2);
     }
   };
 
-  .blur_up {
-    -webkit-filter: blur(6px);
-    filter: blur(6px);
-    transition: filter 550ms, -webkit-filter 550ms;
+  &.fade {
+    opacity: 0;
+    &.lazyloaded {
+      opacity: 1;
+      transition: opacity 300ms ease-in;
+    }
   }
-
-  .blur_up.lazyloaded {
-    -webkit-filter: blur(0);
-    filter: blur(0);
+  
+  &.blur {
+    filter: blur(5px);
+    &.lazyloaded {
+      -webkit-filter: blur(0);
+      filter: blur(0);
+      transition: filter 300ms ease-in;
+    }
+  }
+  
+  &.scale {
+    opacity: 0;
+    transform: scale(0.9);
+    &.lazyloaded {
+      opacity: 1;
+      transform: scale(1);
+      transition: 700ms ease-in;
+    }
```

Comment:
The CSS loading example in the story is different.  We should confirm which one is correct. 
https://nitro.powerhrg.com/runway/backlog_items/PLAY-19

Reply:
I remember we took this css styling from the image.scss file. I would assume we would keep the image styling from an already existing playbook kit. Also im not sure if this is going to make a difference since im not the one to have a final say, but the scale transition and the fade transition is a bit too quick and abrupt for the end user.  For the sake of uniformity across playbook/nitro, i would bet that the reused image kit styling would be a safe bet. I'll ask during dailies

Reply:
We answered this question, we decided to set the load times to 400ms. Also the current code works as is without breaking anything so far... We may also be alpha testing this PR, so we'll be on the lookout if anything breaks with the background images.